src/main/java/org/beta/authentification/keycloak/authenticator/OtpLoginAuthenticator.java

```diff
@@ -104,7 +103,7 @@ public void authenticate(AuthenticationFlowContext context) {
             context.challenge(
                     context.form()
                             .setAttribute(FORM_ATTRIBUTE_USER_EMAIL, user.getUsername())
-                            .setAttribute("feature_tchap_bot", Features.isTchapBotEnabled())
+                            .setAttribute("feature_tchap_bot", Features.isTchapBotEnabled(context.getAuthenticationSession().getClient()))
```

Comment:
can this value be null at any point?

Reply:
I have no idea. Maybe there is a case in the process where this value can be null.